net-proxy/sqtop/Manifest

```diff
@@ -0,0 +1 @@
+DIST v2015-02-08.tar.gz 154535 BLAKE2B 4825a254670e5336acbddc8ffb50c7a423614c06fcf9b69b354e3ccf88496bde17ba586bf45cf062a4da40e2332b84a611b50582e425278a259cc27a23f53f7f SHA512 35911b2d4aaa6364ec52bb55264372378b4a458268cfc2e03884660576014739a55e255eb751a9be54b70d4504f16d095b6cf9db08ff5889d52fd2a46ab2033a
```

Comment:
So this is going to divide people and I'm on the side who don't wish "new" packages to be last-released in 2015. Maybe GURU is a better place for this for now?

Reply:
Package is stable, fit for purpose.  Does what it does well and unless something changes squid side no changes should be required.  Only breakage in the history of this use was a minor tweak required from squid3 to squid4 - else you'd probably be looking at a 2007 package.

We find this extremely useful.  Others may or may not.